test/metabase/api/testing_test.clj

```diff
@@ -54,3 +56,21 @@
     (mt/with-temp-empty-app-db [_conn :h2]
       (#'testing/restore-snapshot! snapshot-name)
       (is (= [{:a 1}] (jdbc/query {:datasource (mdb/app-db)} ["select a from test_view"]))))))
+
+(deftest set-time-test
+  (let [t (t/zoned-date-time 2024 7 8 15 00 00)]
```

Comment:
We should have a `try/finally` to set back the clock to its original value here to prevent leaking this setting.

Reply:
oh right